### DISCUSSION TITLE
Is it "safe" to increase the scrape limit?

### DISCUSSION THREAD

**Discussion by mricsi97** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#257] [Q&A] [answered]
Title. By scrape limit I mean the number of pages the script scrapes from the 3 sources in a single run. Why is the default 5? Is it likely that one of the sites (incl. Udemy) detects the bot if I increase it?

**Comment by fakeid30** [comment, marked as answer]
Default is 5 because after 5 pages you are most likely going to hit links that are expired. 

You can increase the limit for first run.

**Reply by mricsi97** [reply]
I see, thanks!

**Reply by fakeid30** [reply]
Glad to be of help.
